Makefile

```diff
@@ -157,7 +157,14 @@ stylus_benchmarks = $(wildcard $(stylus_dir)/*.toml $(stylus_dir)/src/*.rs) $(st
 
 espresso_crypto_dir = ./espressocrypto/lib/espresso-crypto-helper
 espresso_crypto_files = $(wildcard $(espresso_crypto_dir)/*.toml $(espresso_crypto_dir)/src/*.rs)
-espresso_crypto_lib = $(output_root)/lib/libespresso_crypto_helper.a
+espresso_crypto_lib = $(output_root)/lib/libespresso_crypto_helper
+espresso_crypto_filename = libespresso_crypto_helper.so
+ifeq ($(UNAME_S), Darwin)
+	espresso_crypto_filename = libespresso_crypto_helper.dylib
+else
+	export LD_LIBRARY_PATH="$(pwd)/target/lib:$LD_LIBRARY_PATH"
```

Comment:
I think this should fix the current CI issues but not really know what's happening